pkg/sosreport/sosreport.jsx

```diff
@@ -177,7 +177,9 @@ function sosCreate(args, setProgress, setError, setErrorDetail) {
     });
 
     task.catch(error => {
-        console.error(JSON.stringify(error)); // easier investigation of failures, errors in pty mode may be hard to see
+        // easier investigation of failures, errors in pty mode may be hard to see
+        if (error.problem !== 'cancelled')
+            console.error("Failed to call sos report:", JSON.stringify(error));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19509-20231020-113103-f8bf272a-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)